Doc/Tutorial/chapter_testing.tex

```diff
@@ -272,7 +272,7 @@ \subsection{Writing a unittest-based test}
 The \verb|unittest|-framework has been included with Python since version
 2.1, and is documented in the Python Library Reference (which I know you
 are keeping under your pillow, as recommended).  There is also
-\href{http://docs.python.org/library/unittest.html}{online documentaion
+\href{https://docs.python.org/2/library/unittest.html}{online documentaion
```

Comment:
Again, https://docs.python.org/3/library/unittest.html rather than the Python 2 link please.

Reply:
I use the reference to Python 3 documentation.